src/cpr_data_access/models.py

```diff
@@ -43,6 +43,64 @@
 AnyDocument = TypeVar("AnyDocument", bound="BaseDocument")
 
 
+def _load_and_validate_metadata_csv(
+    metadata_csv_path: Path, target_model: type[AnyDocument]
+) -> pd.DataFrame:
+    """Load a metadata CSV, raising a ValueError if it does not exist or doesn't have the expected columns."""
+    if not metadata_csv_path.exists():
+        raise ValueError(f"metadata_csv_path {metadata_csv_path} does not exist")
+
+    if not metadata_csv_path.is_file() or not metadata_csv_path.suffix == ".csv":
+        raise ValueError(f"metadata_csv_path {metadata_csv_path} must be a csv file")
+
+    metadata_df = pd.read_csv(metadata_csv_path)
```

Comment:
Do we need to think about the sheet that we are reading from and which row on that sheet we are reading from?

Reply:
they're CSVs so no sheets, and the ones i have start on line 0. i've seen others for projects but these won't work with this library